specification/compute/resource-manager/Microsoft.Compute/stable/2018-06-01/compute.json

```diff
@@ -5195,7 +5198,7 @@
         "Local"
       ],
       "x-ms-enum": {
-        "name": "DiffDiskOption",
+        "name": "DiffDiskOptions",
```

Comment:
This will introduce a breaking change.  Looks like it can't be avoided though due to the name collision?

Reply:
Oh nevermind this is part of the original commit.